### DISCUSSION TITLE
Finding BBox of the tracked object

### DISCUSSION THREAD

**Discussion by muhammadanas25** [tryolabs/norfair#255] [Q&A] [answered]
Dear Authors,
I am trying to retrieve the bounding box of the tracked object, but I was unable to find a way to do it when going through the code base. I noticed an internal variable, initial detection, that seems to contain the necessary information, but I do not have access to it through the TrackedObject instance.

self.dim_points = initial_detection.absolute_points.shape[1]
self.num_points = initial_detection.absolute_points.shape[0]

I am looking for a way to save the crops of the tracked objects to a relative destination without having to clone the entire repository and make manual changes.

Thank you for your time and consideration.

Best regards,
M. Anas

**Comment by facundo-lezama** [comment, marked as answer]
Hi @muhammadanas25, 

In `tracked_object.last_detection`, you can find the last detection that was matched with the `tracked_object`. But if you need to access the estimated position of the bounding box as calculated by the Kalman filter, then you can get it from `tracked_object.get_estimate()`.
